Doc/whatsnew/3.2.rst

```diff
@@ -1060,15 +1060,15 @@ of nearly equal quantities:
 
 The :func:`~math.erf` function computes a probability integral or `Gaussian
 error function <https://en.wikipedia.org/wiki/Error_function>`_.  The
-complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``:
+complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``::
```

Comment:
@bitdancer thank you very much for the explanation :-)  I hope at the end of this task we will write some guidelines for the doctest, in order to have only one way to write them.  In this commit I deindend the code in order to allign it to the doctest directive.  @berkerpeksag for sure I am not able to do any lecture to you :D  If you misunderstand it is becose of my English, I suppose.  Sorry :(   Thank you very much for all the time you are spending in these reviews and answering my questions :-)